lms/djangoapps/courseware/access.py

```diff
@@ -270,17 +270,22 @@ def _can_enroll_courselike(user, courselike):
     return ACCESS_DENIED
 
 
-def _has_access_course_desc(user, action, course):
+def _has_access_courselike(user, action, courselike):
     """
-    Check if user has access to a course descriptor.
+    Check if user has access to a course.
+
+    Arguments:
+        user (User): the user whose course access we are checking.
+        action (string): The action that is being checked.
+        courselike (CourseDescriptor or CourseOverview): The object
+            representing the course that the user wants to access.
```

Comment:
I'd prefer the method name: _has_access_course (or _has_access_to_course).
I don't love the name "courselike", but if it must stay, I think it is only needed in the param.
Additionally, could the param be "course_desc_or_overview"?  Anything else I can think of is equally vague, and might as well go with "courselike" if you don't like my suggestion.  Also, is "like" a common suffix for this type of variable in Python?